defaults/main.yml

```diff
@@ -163,6 +163,13 @@ datadog_windows_config_root: "{{ ansible_facts.env['ProgramData'] }}\\Datadog"
 # arguments to supply to the windows installer.
 win_install_args: " "
 
+# A boolean to enable host injection.
+datadog_apm_host_injection_enabled: false
+# A boolean to enable docker injection.
+datadog_apm_docker_injection_enabled: false
+# List of apm libraries to install if host or docker injection is enabled.
+# Valid values are: java, js, dotnet, all
```

Comment:
I think we have `python` now as well, right?

I think in general this comment should contain a note "as of writing this comment", because the set of libraries might change in time which could be confusing to users with older version of this role.